make/photon/db/registry.sql

```diff
@@ -272,6 +272,20 @@ create table harbor_label (
  CONSTRAINT unique_name_and_scope UNIQUE (name,scope)
  );
 
+create table harbor_resource_label (
+ id int NOT NULL AUTO_INCREMENT,
+ label_id int NOT NULL,
+ resource_id varchar(256) NOT NULL,
```

Comment:
Could you specify what the `resource_id` would look like for different `resource_type`?